ivtest/regress-fsv.list

```diff
@@ -82,6 +82,7 @@ pr3015421		CE			ivltests gold=pr3015421-fsv.gold
 resetall		normal,-Wtimescale	ivltests gold=resetall-fsv.gold
 scope2b			normal			ivltests
 sys_func_task_error	RE			ivltests gold=sys_func_task_error-fsv.gold
+unnamed_block_var_decl	CO			ivltests
```

Comment:
I would like to see this changed to "normal", and at least a stub $display("PASSED") in the test program.

Reply:
Ok, added that.